.doc_gen/metadata/iot_metadata.yaml

```diff
@@ -429,10 +429,8 @@ iot_CreateThing:
   services:
     iot: {CreateThing}
 iot_Scenario:
-  title: Work with &IoT; devices, things, and shadows using &IoT; SDK
-  title_abbrev: Work with device management use cases
-  synopsis: work with &IoT; device management use cases using &IoT; SDK
-  category: Scenarios
+  synopsis: work with &IoT; device management.
```

Comment:
This example might better be explained with a synopsis list.

Reply:
I don't disagree, but this is out of the scope of this project so we'll have to tackle rewrites in the future.